recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "xsimd" %}
-{% set version = "7.5.1" %}
+{% set version = "7.5.0" %}
 {% set sha256 = "a865a83967161069b0eaa142818480384f82b818489f6d2187847c6ebf6d8d44" %}
```

Comment:
These need to be updated in sync...

Reply:
Though there shouldn't be a reason to downgrade the version?

Reply:
Yes, I forgot we had a broken release of xsimd, thus the update of the version number. I will wait for the bot to open the PR for the arm build and will fix everything in it.